frontend/src/modules/app/ToolBar.tsx

```diff
@@ -397,30 +396,16 @@ export const ToolBar = ({
         onClose={handleDiscardClose}
         onConfirm={handleDiscardConfirm}
       >
-        <Grid container alignItems="center">
-          <Grid container alignItems="center" justify="center">
-            <Grid container alignItems="center" className={classes.marginHeader}>
-              <Grid item xs>
-                <Typography variant="h4">Keep Previous Values?</Typography>
-              </Grid>
-            </Grid>
-            <Grid item className={classes.marginContent}>
-              {alarmConfig.map((param: AlarmConfiguration) => {
-                if (alarmLimitsRequest !== null && alarmLimitsRequestDraft !== null) {
-                  if (alarmLimitsUnsavedKeys.includes(param.stateKey)) {
-                    return (
-                      <Typography variant="subtitle1">{`Keep ${param.label} alarm range to ${
-                        alarmLimitsRequest[param.stateKey].lower
-                      } -
-                                ${alarmLimitsRequest[param.stateKey].upper}?`}</Typography>
-                    );
-                  }
-                }
-                return <React.Fragment />;
-              })}
-            </Grid>
-          </Grid>
-        </Grid>
+        <ModalContent />
+      </ModalPopup>
+      <ModalPopup
```

Comment:
Is there any way to just use one ModalPopup? If we're going to decompose the Start/Pause Ventilation button out anyways, then it could make sense to keep two ModalPopups (as one ModalPopup would be moved out of the Toolbar component and into the Start/Pause Ventilation component)

Reply:
So the problem here really is in the `onClick` handler and the `open` attribute which decides when the modal should open and those two behaviours need to be distinct which is why two ModalPopup